deployer/src/config_creator.py

```diff
@@ -68,6 +72,38 @@ def to_docusaurus_config(config, urls=None):
     return config
 
 
+def to_docusaurus_v2_config(config, urls=None):
+    assert_list_non_empty(urls)
+    if urls:
+        config["sitemap_urls"] = [
+            extract_root_from_input(urls[0]) + "sitemap.xml"]
+        config["sitemap_alternate_links"] = True
+    start_url = urls[0]
+    if '/docs/' not in start_url:
+        if not start_url.endswith('/'):
+            start_url = start_url + '/'
+
+        config["start_urls"] = [start_url + 'docs/']
+    else:
+        config["start_urls"] = [start_url]
```

Comment:
As in `to_docusaurus_config` above:
```suggestion
        start_url += 'docs/'
    config["start_urls"] = [start_url]
```

Reply:
Good call. Resolved